bot/EchoWithCounterBot.cs

```diff
@@ -259,18 +256,11 @@ private HashSet<string> ExtractAndDefineCryptonyms(string question)
         // Both Key Phrases and Entities are extracted
         private async Task<HashSet<string>> ExtractKeywords(string question, HashSet<string> terms)
         {
-            // use Text Analytics to get key phrases and entities from the question, which will become the search query
-            // we will use the same Text Analytics query for both key words and entities
-            var doc = new MultiLanguageBatchInput(
-                new List<MultiLanguageInput>()
-                {
-                    new MultiLanguageInput("en", "0", question),
-                });
 
             // do the Text Analytics requests asynchronously so both can proceed at the same time
             logger.LogTrace($"HOOVERBOT {turnid} making Text Analytics requests.");
-            var t_keyphrases = textAnalyticsClient.KeyPhrasesAsync(doc);
-            var t_entities = textAnalyticsClient.EntitiesAsync(doc);
+            var t_keyphrases = textAnalyticsClient.ExtractKeyPhrasesAsync(question);
+            var t_entities = textAnalyticsClient.RecognizeEntitiesAsync(question);
             var keyphrases = await t_keyphrases;
             var entities = await t_entities;
```

Comment:
```suggestion
            var keyphrases = await textAnalyticsClient.ExtractKeyPhrasesAsync(question);
            var entities = await textAnalyticsClient.RecognizeEntitiesAsync(question);
```
There's no benefit to capturing the task to await it on the next line.   We should use the normal async call pattern.